agenda/presentation/src/main/java/com/nibalk/tasky/agenda/presentation/home/HomeScreen.kt

```diff
@@ -26,78 +33,85 @@ import com.nibalk.tasky.agenda.presentation.model.AgendaItemActionType
 import com.nibalk.tasky.agenda.presentation.model.AgendaType
 import com.nibalk.tasky.agenda.presentation.utils.getSurroundingDays
 import com.nibalk.tasky.core.presentation.components.TaskyBackground
+import com.nibalk.tasky.core.presentation.components.TaskyEmptyList
+import com.nibalk.tasky.core.presentation.components.TaskyNeedleSeparator
 import com.nibalk.tasky.core.presentation.themes.TaskyTheme
+import com.nibalk.tasky.core.presentation.themes.spacing
 import com.nibalk.tasky.core.presentation.utils.ObserveAsEvents
 import org.koin.androidx.compose.koinViewModel
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+import com.nibalk.tasky.core.presentation.R as CoreR
+
+typealias IsDetailScreenEditable = Boolean
 
 @Composable
 fun HomeScreenRoot(
-    onDetailClicked: (Boolean) -> Unit,
+    onDetailClicked: (IsDetailScreenEditable, AgendaType, AgendaItem?) -> Unit,
```

Comment:
You don't need to create a typealias here, you can give lambda parameters a name:
```
onDetailClicked: (isDetailScreenEditable: Boolean, AgendaType, AgendaItem?) -> Unit
```